src/modules/45-projects-orgs/AdminRouteDestinations.tsx

```diff
@@ -56,6 +56,9 @@ const AdminRouteDestinations = (): React.ReactElement => {
       >
         <AdminRedirect />
       </RouteWithContext>
+      <RouteWithContext exact path={routes.toLandingDashboard({ ...accountPathProps })}>
+        <Redirect to={routes.toOverview()} />
+      </RouteWithContext>
       <RouteWithContext path={routes.toOverview({ ...accountPathProps, ...modePathProps })}>
         <LandingDashboardPage />
       </RouteWithContext>
```

Comment:
```suggestion
      <RouteWithContext path={[routes.toLandingDashboard({ ...accountPathProps, ...modePathProps }), routes.toOverview({ ...accountPathProps, ...modePathProps })]}>
        <LandingDashboardPage />
      </RouteWithContext>
```

This should also do the trick

Reply:
This won't work as we need redirection from `get-started` to `overview` and not just rendering of page